internal/handler/awsmapping.go

```diff
@@ -66,6 +66,12 @@ func AwsMappingHandler(ctx context.Context) error {
 			return nil
 		default:
 		}
+
+		if resp.Account.Status == "SUSPENDED" {
```

Comment:
Can you extract "SUSPENDED" to a constant

Reply:
https://github.com/dfds/aad-aws-sync/pull/31/commits/f87996060998563ed5555f6b12a56037a444ede3